packages/ifpack2/src/Ifpack2_LocalSparseTriangularSolver_def.hpp

```diff
@@ -425,6 +425,10 @@ compute ()
     storeRow( lclNumRows-lclRow-1, newval, newcol );
   }
 
+  // create a new local_matrix in the A_crs_ object
+  // typename Matrix::local_matrix_type A2
+  // A2 = KokkosSparse( "reversed", newval, newrow, newind )
```

Comment:
Please don't use `KokkosSparse` as a typedef.  It's a namespace.  That could confuse the compiler.  Use the existing typedef `local_matrix_type`.